implementations/node/src/api/controllers/_dev/index.ts

```diff
@@ -0,0 +1,30 @@
+import * as Router from 'koa-router';
+
```

Comment:
I don't think we need methods for static files like this. There's a koa-static plugin. I wasn't able to test because I couldn't get the playground running, but here's a diff of what I think can be made to work:
```
diff --git a/implementations/node/src/api/controllers/_dev/index.ts b/implementations/node/src/api/controllers/_dev/index.ts
index b897deb..2b6ecd6 100644
--- a/implementations/node/src/api/controllers/_dev/index.ts
+++ b/implementations/node/src/api/controllers/_dev/index.ts
@@ -1,30 +1,8 @@
 import * as Router from 'koa-router';

-const fs = require('fs');
+const serve = require('koa-static');
 const devRouter = new Router();

-devRouter.get('/playground', async ctx => {
-  await new Promise((resolve, reject) => {
-    fs.readFile('./test/playground/index.html', function(error, content) {
-      console.log(error);
-      if (error) reject();
-      ctx.set('Content-Type', 'text/html');
-      ctx.body = content;
-      resolve();
-    });
-  });
-});
-
-devRouter.get('/playground/playground.css', async ctx => {
-  await new Promise((resolve, reject) => {
-    fs.readFile('./test/playground/playground.css', function(error, content) {
-      console.log(error);
-      if (error) reject();
-      ctx.set('Content-Type', 'text/css');
-      ctx.body = content;
-      resolve();
-    });
-  });
-});
+devRouter.use(serve('./test/playground'));
```